Frameworks/UIKit/StarboardXaml/StoryboardManager.cpp

```diff
@@ -148,6 +146,16 @@ void StoryboardManager::_CreateFlip(Layer^ layer, bool flipRight, bool invert, b
                 unsigned int index = 0;
                 parentPanel->Children->IndexOf(layer, &index);
                 parentPanel->Children->RemoveAt(index);
+            } else {
+                // TODO: Track down why we sometimes don't have a parent when the animation completes.
+                TraceWarning(TAG, L"Failed to remove flip animation from scene; didn't have a parent to remove from!");
+                layer->Visibility = Visibility::Collapsed;
+                LayerCoordinator::SetContent(
+                    layer,
+                    nullptr,
+                    static_cast<float>(0),
+                    static_cast<float>(0),
+                    static_cast<float>(0));
```

Comment:
note: still doesn't seem to catch *all* cases; I think we need to do some cleanup when animations are stopped.  i'll follow up further tomorrow.